com.skewjo.dota_match_finder.json

```diff
@@ -0,0 +1,9 @@
+{   
+  "name": "com.skewjo.dota_match_finder",   
+  "description": "Identifies 10 digit numbers as Dota match IDs and turn them into clickable links that download the replay and start it in Dota",
+  "path": "C:\\Users\\LeoRocha\\DotaMatchFinder\\run.bat",
```

Comment:
Any way for us to make this path relative?

Reply:
We need to create an installer and make the user put the right path.